### PR TITLE
Create README on tests for FAN_TRAY type.

### PR DESCRIPTION
**Pull request by SydneyCaulfeild**

### CODE REVIEW

File: feature/gnmi/otg_tests/telemetry_fan_tray_type_test/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+## OpenConfig Path and RPC Coverage
+
+The below yaml defines the OC paths intended to be covered by this test.  OC paths used for test setup are not listed here.
+
+```yaml
+paths:
+  ## State Paths ##
+  /components/component/state/type:
+
```
Comment (dplore) [inline feature/gnmi/otg_tests/telemetry_fan_tray_type_test/README.md:21]:
```suggestion
paths:
  ## State Paths ##
  /components/component/state/type:
    platform_type: ["FAN_TRAY"]
```

File: feature/gnmi/otg_tests/telemetry_fan_tray_type_test/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  gnmi:
+    gNMI.Subscribe:
+        Mode: [ "ON_CHANGE", "ONCE" ]
+    gNMI.Get:
+```
+
+
+## Minimum DUT platform requirement
+
+MMF
```
Comment (dplore) [inline feature/gnmi/otg_tests/telemetry_fan_tray_type_test/README.md:32]: I think we should re-iterate that the modular form factor device must contain FAN_TRAY type hardware.  We can leave it to the test runner (out of scope from featureprofiles testing) to pick/choose the hardware.

**Comment by LimeHat** [comment]
I would suggest updating gnmi-1.4 instead.
https://github.com/openconfig/featureprofiles/tree/main/feature/platform/tests/telemetry_inventory_test

File: feature/gnmi/otg_tests/telemetry_fan_tray_type_test/README.md (hunk 2 above)
Comment (SydneyCaulfeild) [reply feature/gnmi/otg_tests/telemetry_fan_tray_type_test/README.md]: Added this note to the platform requirements.

**Comment by dplore** [comment]
Closed in favor of https://github.com/openconfig/featureprofiles/pull/3046

**Pull request closed without merging**